DuckDuckGo/FilePreviewHelper.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FilePreviewHelper.swift
+//  DuckDuckGo
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+struct FilePreviewHelper {
+    
+    static func fileHandlerForDownload(_ download: Download, viewController: UIViewController) -> FilePreview? {
+        guard let filePath = download.location else { return nil }
+        switch download.mimeType {
+        case .usdz, .reality:
+            return QuickLookPreviewHelper(filePath, viewController: viewController)
+            
+        case .passbook:
+            return PassKitPreviewHelper(filePath, viewController: viewController)
+            
+        default:
+            return nil
```

Comment:
Maybe we want to also use `QuickLookPreviewHelper` in the `default` scenario?

Reply:
OK, I'll remove the `case .usdz, .reality:` then to make it easier to read.